### ISSUE TITLE
[FEAT]ARMOR LIMIT

### ISSUE THREAD

**Issue by chycken** [CS-GEJMERZY/VIP-Plugin#68]
Hi, can you please add an armor limit to the limits? Because currently when armor regenerates, it can go up to 600. Thanks!!
@HackerDevs

**Comment by chycken** [comment]
+ one issue: flags arent working, plugin doesnt add flags to the vips.... @HackerDevs

**Comment by chycken** [comment]
@wohahobg Hey sorry for ping you. Will be this plugin updated, or not supported anymore?

**Comment by wohahobg** [comment]
@chycken hi, sorry for the late response

I been working on an update this night, hope i will push it soon.

Also for this "bug" can you please explain more so I can have a look

**Comment by chycken** [comment]
> [@chycken](https://github.com/chycken) hi, sorry for the late response
> 
> I been working on an update this night, hope i will push it soon.
> 
> Also for this "bug" can you please explain more so I can have a look

When you create a vip group, and put flags onto this group, plugin wont add these flags to the player automatically. You have to run the css_vp_player_addflags command for this. So plugin should add the flags we write to the given group (in the cfg) to the vip player, and we wont have to run that command for it when we add a vip player..

**Comment by wohahobg** [comment]
Hi,

the plugin does not have feature to adding "flags" as you want. 

What I am suggesting you is this instant of group to the user, add flags.  I assume you are talking about  ` "Permissions": "@vip-plugin/vip",` but that's only for one permission to detect the vip group.


However i might make an update to add such as feature like separate module to add "flags" from the group it self.

**Comment by chycken** [comment]
> Hi,
> 
> the plugin does not have feature to adding "flags" as you want.
> 
> What I am suggesting you is this instant of group to the user, add flags. I assume you are talking about ` "Permissions": "@vip-plugin/vip",` but that's only for one permission to detect the vip group.
> 
> However i might make an update to add such as feature like separate module to add "flags" from the group it self.
Yeah that what i wanted to say, so if i add 3 flags to the given vip group and add a player for this vip group, then this player will automatically have these 3 flags.. thanks.
